C#/Botkit/botbuilder-slack-adapter/DialogData.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace botbuilder_slack_adapter
+{
+    public class DialogData
+    {
+        public string Title { get; set; }

```

Comment:
Should all these properties be public? In the [original code](https://github.com/howdyai/botkit/blob/master/packages/botbuilder-adapter-slack/src/slack_dialog.ts#L21) `data` was private, and its properties were assigned using the constructor.

Reply:
We need them to be public so they can be set from SlackDialog class. Thanks!